docs/addon/walkthrough/README.md

```diff
@@ -316,80 +329,82 @@ we'll need a Docker image and some manifests.
 1. Modify the IMG value in the `Makefile` to reflect a docker registry that you 
    can write to:
 
-   ```make
-   # Image URL to use all building/pushing image targets
-   IMG ?= gcr.io/<my-cool-project>/dashboard-operator:latest
-   ```
+```make
+# Image URL to use all building/pushing image targets
+IMG ?= gcr.io/<my-cool-project>/dashboard-operator:latest
+```
 
 1. Create a patch to modify the memory limit for the operator:
-   ```bash
-   echo << EOF > config/default/manager_resource_patch.yaml
-	apiVersion: apps/v1
-	kind: StatefulSet
-	metadata:
-	  name: controller-manager
-	  namespace: system
-	spec:
-	  template:
-	    spec:
-	      containers:
-	      - name: manager
-	        resources:
-	          limits:
-	            cpu: 100m
-	            memory: 150Mi
-	          requests:
-	            cpu: 100m
-	            memory: 20Mi
-   EOF
-   ```
+
+```bash
+echo << EOF > config/default/manager_resource_patch.yaml
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
The kind must be changed to `Deployment` here.
The format change makes it possible to copy-and-paste of this snippet.